WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -43,12 +36,8 @@ public WalletViewModel(UiContext uiContext, IWalletModel walletModel, Wallet wal
 
 		_title = WalletName;
 
-		// TODO:
-		//this.WhenAnyValue(x => x.IsCoinJoining)
-		//	.Skip(1)
-		//	.Subscribe(_ => MainViewModel.Instance.InvalidateIsCoinJoinActive());
-
-		Disposables = Disposables is null
+		Disposables =
```

Comment:
The has not been disposed since the beginning I think.

Reply:
You're right. It never gets disposed of, because we don't support unloading of loaded wallets. Should I remove the Disposable entirely for now?

Reply:
Yes, please